plugins/github-pull-requests-board/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@backstage/plugin-github-pull-requests-board",
+  "description": "A Backstage plugin that allows you to see all open Pull Requests for all the repositories owned by your team",
+  "version": "0.1.0",
```

Comment:
as a new plugin this should start on `0.0.0`. The changeset will make sure the first version is `0.1.0.`